src/libstd/thread/mod.rs

```diff
@@ -714,33 +721,21 @@ struct Inner {
 
 #[derive(Clone)]
 #[stable(feature = "rust1", since = "1.0.0")]
-/// A handle to a thread.
+/// Threads are represented via the `Thread` type, which you can get in one of
+/// two ways:
```

Comment:
This first line needs to be a summary; this doesn't quite work. I think leaving the first line, adding a `///` in between, and keeping this new stuff is 👍